UpToDate References.js

```diff
@@ -36,48 +36,49 @@
 	***** END LICENSE BLOCK *****
 */
 
-function detectWeb(doc, url) {
+function detectWeb(doc) {
```

Comment:
I would prefer to use here the `detectWeb(doc, _url)` variable, because it may be that in some later changes we want to start using that variable.